v1/embedded/developer/intro.mdx

```diff
@@ -0,0 +1,129 @@
+---
+title: Introduction
+---
+
+This introduction includes high-level instructions on building the front-end component of your integration hub within
+your own application.
+
+## Choosing your embedded implementation
+
+Before continuing, it's worth understanding two approaches to building your integration hub.
+
+### Full Client-Side
+
+In this approach, your integration hub is built entirely in the client-side code of your application. This is the
+simplest (and quickest) way to get started for most applications, however some users may prefer implementing some
+server-side logic to facilitate the integration hub, in which case the hybrid approach is available.
+
+#### Authentication
+
+When using the fully client-side approach, you will need to authenticate your users against Versori APIs using [Signing
+Key][signing-keys] tokens.
+
+<Warning>
+    You should never use [API Keys][api-keys] with the full client-side approach. These keys have full access to your
+    organisation and should not be exposed to your users in any way.
+
+    Nor should you expose your Signing Key to your users. This key should be kept secure and only used in your
+    server-side code to issue JWT tokens.
+
+</Warning>
+
+This approach involves implementing a new API endpoint in your own application which can authenticate your user and
+issue a JWT token signed with a [Signing Key][signing-keys]. This token can then be used to authenticate your user
+against Versori APIs, and they will only have access to manage their own activations to your integration hub.
+
+<Info>
+    We are working on providing reference implementations for this endpoint in multiple languages/frameworks. If you
+    would like to discuss this further, please reach out to us.
+</Info>
+
+### Hybrid Client-Side and Server-Side
+
+In this approach, your integration hub is built using a combination of client-side and server-side code. This allows you
+to pick and choose which parts of the integration hub are built in the client-side versus the server-side to suit your
+application's requirements.
+
+#### Authentication
+
+If communicating with Versori APIs from your server-side code, you can use [API Keys][api-keys] or [Signing
+Key][signing-keys] tokens, but it is your responsibility to authenticate users at your own API boundary. However, as
```

Comment:
See above, although in this case I'm pretty sure its an End User here. Tbh it is cumbersome to use End User all the time though